components/src/AppHeaderTop/appheadertop.main.tsx

```diff
@@ -57,13 +60,25 @@ class AppHeaderTop extends React.Component<AppHeaderTopProps> {
       appHeaderTopLinks,
     } = this.props;
 
+    const impersonating = localStorage.getItem(`${Config.cortexApi.scope}_oAuthImpersonationToken`);
+    const userName = localStorage.getItem(`${Config.cortexApi.scope}_oAuthUserName`) || localStorage.getItem(`${Config.cortexApi.scope}_oAuthUserId`);
+
     return [
       <div key="AppHeaderTop" className={`top-header ${isMobileView ? 'mobile-view' : ''}`}>
-        <div className="top-container">
+        <div className={impersonating ? 'top-container-impersonation' : 'top-container'}>
           <div className="locale-container">
             <AppHeaderLocaleMain onCurrencyChange={onCurrencyChange} onLocaleChange={onLocaleChange} />
           </div>
 
+          {
+            impersonating ? (
```

Comment:
If this is meant to persist on the page, can you please provide a screenshot of this notification?

Reply:
Posted screenshots in the description of the impersonation header.